### PR TITLE
Update Description

### PR DESCRIPTION
**Pull request by arcadiafalcone**
## Why was this change made?

Updates to Description schema

## How was this change tested?

Online Swagger editor

## Which documentation and/or configurations were updated?

cocina-descriptive-metadata

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -435,6 +436,7 @@ components:
               description: Status of the descriptive element value relative to other instances
                 of the element.
               type: string
+              # type: integer
```
Comment (justinlittman) [inline openapi.yml:439]: Cruft?

File: openapi.yml [hunk 2]
```diff
@@ -950,6 +972,9 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/DescriptiveValue"
+        standard:
+          #description: Descriptive or content standards to which the value conforms.
```
Comment (justinlittman) [inline openapi.yml:976]: Cruft?

File: openapi.yml [hunk 3]
```diff
@@ -1263,8 +1291,8 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/DescriptiveValue"
-        source:
-          $ref: "#/components/schemas/Source"
         version:
-          description: The version of the standard or encoding.
+          description: The version of the value standard.
```
Comment (justinlittman) [inline openapi.yml:1295]: What is a value standard?

**Review by justinlittman — CHANGES_REQUESTED** [review]

File: openapi.yml (hunk 2 above)
Comment (jcoyne) [reply openapi.yml:976]: No, you can't provide description with a $ref. So this is just an annotation for humans.

Comment (justinlittman) [reply openapi.yml:976]: OK. Recommend `# description` to make it seem a bit less crufty but not a big deal.

File: openapi.yml (hunk 1 above)
Comment (arcadiafalcone) [reply openapi.yml:439]: See #154 . It's in the conceptual model but can't yet be implemented due to a bug elsewhere.

File: openapi.yml (hunk 3 above)
Comment (arcadiafalcone) [reply openapi.yml:1295]: Value standard is about how you derive and construct the metadata values. For example, MARC says the title goes in 245$a, but it's the content standard RDA that has rules about where you get the title from (cover? title page? spine?), how you transcribe it, what kinds of title variations should be included, how to handle a typo on the resource, etc. Other examples of library value standards are DACS (archives) and the various RBMS standards (rare books and manuscripts).

File: openapi.yml (hunk 1 above)
Comment (justinlittman) [reply openapi.yml:439]: Can you add that to the comment?

File: openapi.yml (hunk 3 above)
Comment (justinlittman) [reply openapi.yml:1295]: Can you add `e.g., RDA`? At least for me, that would make it much clearer.

**Review by justinlittman — APPROVED** [review]

**Pull request merged**